exercises/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,207 @@
+        "base": 10
+      },
+      "expected": false
+    },
+    {
+      "description": "100 is not an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 100,
+        "base": 10
+      },
+      "expected": false
+    },
+    {
+      "description": "111 is not an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 111,
+        "base": 10
+      },
+      "expected": false
+    },
+    {
+      "description": "153 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 153,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "370 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 370,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "9474 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 9474,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "0 is an Armstrong number in base 8",
+      "property": "armstrong",
+      "input": {
+        "number": 0,
+        "base": 8
+      },
+      "expected": true
+    },
+    {
+      "description": "1 is an Armstrong number in base 8",
+      "property": "armstrong",
+      "input": {
+        "number": 1,
+        "base": 8
+      },
+      "expected": true
+    },
+    {
+      "description": "2 is an Armstrong number in base 8",
+      "property": "armstrong",
+      "input": {
+        "number": 2,
+        "base": 8
+      },
+      "expected": true
+    },
+    {
+      "description": "7 is an Armstrong number in base 8",
+      "property": "armstrong",
+      "input": {
+        "number": 7,
+        "base": 8
+      },
+      "expected": true
+    },
+    {
+      "description": "10 is not an Armstrong number in base 8",
+      "property": "armstrong",
+      "input": {
+        "number": 10,
+        "base": 8
+      },
+      "expected": false
+    },
+    {
+      "description": "432 is an Armstrong number in base 8",
+      "property": "armstrong",
+      "input": {
+        "number": 432,
+        "base": 8
+      },
+      "expected": true
+    },
+    {
+      "description": "2 is an Armstrong number in base 3",
+      "property": "armstrong",
+      "input": {
+        "number": 2,
+        "base": 3
+      },
+      "expected": true
+    },
+    {
+      "description": "4 is not an Armstrong number in base 3",
+      "property": "armstrong",
+      "input": {
+        "number": 4,
+        "base": 3
+      },
+      "expected": false
+    },
+    {
+      "description": "17 is an Armstrong number in base 3",
+      "property": "armstrong",
+      "input": {
+        "number": 17,
+        "base": 3
+      },
+      "expected": true
+    },
+    {
+      "description": "10 is not an Armstrong number in base 8",
```

Comment:
Isn't this a duplicate of [L135](https://github.com/exercism/problem-specifications/pull/1018/files#diff-e2fa98853c36612fa9ec636bd3055158R135)?

Reply:
Removed, Thank you